### ISSUE TITLE
Reduce operators causes trouble with multiple decorators

### ISSUE THREAD

**Issue by sdekrijger** [liftoff/pyminifier#34]
Given this test code:
```python
from functools import wraps

def my_decorator(extra_value=None):
    def _my_decorator(view_func):
        def _decorator(request, *args, **kwargs):
            # maybe do something before the view_func call
            # that uses `extra_value` and the `request` object
            response = view_func(request, *args, **kwargs)
            # maybe do something after the view_func call
            return response
        return wraps(view_func)(_decorator)
    return _my_decorator

class Test(object):
  
  @my_decorator()
  @my_decorator()
  def foo(request): 
    return 'yes'
```
The output causes an IndentationError :

```python
#!/usr/bin/python
from functools import wraps
def my_decorator(extra_value=None):
 def _my_decorator(view_func):
  def _decorator(request,*args,**kwargs):
   response=view_func(request,*args,**kwargs)
   return response
  return wraps(view_func)(_decorator)
 return _my_decorator
class Test(object):
 @my_decorator()
@my_decorator()
 def foo(request):
  return 'yes'
# Created by pyminifier (https://github.com/liftoff/pyminifier)
```
I tracked it down to the reduce_operators function, seems that tokens and strings are being mixed up (there is no untokenize call, and out_tokens is not used) ?

**Comment by liftoff** [comment]
Definitely a bug.  I'll fix it :smile:

**Comment by liftoff** [comment]
That was easy.  Can you grab the latest code and confirm that I fixed it?  Thanks.

**Comment by sdekrijger** [comment]
Confirmed fixed issue #33 and #34. Thanks.
